src/logger/POSTLogger.ts

```diff
@@ -91,7 +108,7 @@ export default class MeetingSessionPOSTLogger implements Destroyable {
     return this.logCapture.length;
   }
 
-  startLogPublishScheduler(batchSize: number): void {
+  start(): void {
```

Comment:
Should this be private?

Reply:
It seems this class was implemented to help builders start stop the logger. I have kept the internal working same as is so dint touch these existing APIs. I removed the `batchSize` as it is already a class member variable which `start` can implicitly use.

Do you still want to make them private?

Reply:
Just nit comment so up to you but does not seem like builders want to call start since it is already called in constructor.

Reply:
But I agree that for the POSTLogger both `start` and `stop` can be private and if builders want to stop logging, then they have to just destroy the logger can create a new one.